runtime-testsuite/test/org/antlr/v4/test/runtime/descriptors/trans.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
I think this script is excess in the repository because it's being used only one time for conversion.